deps/build.sh

```
@@ -0,0 +1,15 @@
+#!/bin/sh
```

should add a `set -e` here so the script fails early if any step fails